tests/special_tests.sh

```diff
@@ -12,6 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # Running special tests
+set -e
```

Comment:
what does this do?

Reply:
I noticed special_tests.sh doesn't return an error when a test fails. I found set -e might.